src/routes/Student.js

```diff
@@ -104,7 +113,7 @@ router.route('/add').post( async (req, res) => {
     const faculty = req.body.faculty;
     const privateProfile= req.body.privateProfile;
 
-    const newUser = new User({
+    const newStudent = new Student({
```

Comment:
student creation responsability should not be in the route. We can define a student factory in another file that will create students based on the data passed in the request. We want low coupling here.